Rename security-pack to security-bundle (#78)
giantswarm/security-bundle: CHANGELOG.md

## [Unreleased]

### Changed

- Renamed `security-pack` to `security-bundle`.
- Renamed `kyverno-policies` value to `kyvernoPolicies`.

> - Removed `security-bundle` from the `playground` catalog. Users need to make sure they are using the `giantswarm` catalog.

Reviewer: ```suggestion
- Removed `security-bundle` from the `playground` catalog. **Users must now install `security-bundle` from the `giantswarm` catalog**.
```

> - Renamed `security-pack` to `security-bundle`.
> - Renamed `kyverno-policies` value to `kyvernoPolicies`.
> - Removed `security-bundle` from the `playground` catalog. Users need to make sure they are using the `giantswarm` catalog.

Reviewer: ```suggestion
- Renamed `security-pack` to `security-bundle`.
- Changed default installation namespace from `security-pack` to `security-bundle`.
- Renamed `kyverno-policies` value to `kyvernoPolicies`.
- Removed `security-bundle` from the `playground` catalog. Users need to make sure they are using the `giantswarm` catalog.
```
